packages/kit/src/runtime/server/page/load_node.js

```diff
@@ -98,8 +98,12 @@ export async function load_node({
 				let response;
 
 				if (/^[a-zA-Z]+:/.test(url)) {
-					// external fetch
-					response = await fetch(url, /** @type {RequestInit} */ (opts));
+					const request = new Request(url, /** @type {RequestInit} */ (opts));
```

Comment:
Is a `Request` object really present in all environments and not only node?

Reply:
This is `load_node.js` should be represented server-side only. (I hope NodeJS only)

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Request should be supported very well anyway.

Reply:
@dummdidumm Adapters are required to make `Request` globally available if it isn't already part of the target platform. It's part of the (currently undocumented) contract — see e.g. adapter-node https://github.com/sveltejs/kit/blob/babe515f9f5c816fd507920f1f16012263e7a71f/packages/adapter-node/src/server.js#L8